scipy/stats/_continuous_distns.py

```diff
@@ -4076,6 +4076,101 @@ def fit(self, data, *args, **kwds):
 laplace = laplace_gen(name='laplace')
 
 
+class laplace_asymmetric_gen(rv_continuous):
+    r"""An asymmetric Laplace continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `laplace_asymmetric` is
+
+    .. math::
+
+        f(x, \kappa) &= \frac{1}{\kappa+\kappa^{-1}}\exp(-x\kappa),\quad x\ge0\\
+                     &= \frac{1}{\kappa+\kappa^{-1}}\exp(x/\kappa),\quad x<0\\
+
+    for :math:`-\infty < x < \infty`, :math:`\kappa > 0`.
+
+    `laplace_asymmetric` takes ``kappa`` as a shape parameter for
+    :math:`\kappa`. For :math:`\kappa = 1`, it is identical to a
+    Laplace distribution.
+
+    %(after_notes)s
+
+    See Also
+    --------
+    laplace : Laplace distribution
+
+    References
+    ----------
+    .. [1] Kozubowski TJ and Podg\'orski K. A Multivariate and
```

Comment:
Use Unicode here:
```suggestion
    .. [1] Kozubowski TJ and Podgórski K. A Multivariate and
```
We have a restricted subset of Unicode characters that we allow in the docstrings, and currently ó is not in that set. It is a widely used latin-1 character, though, so we can add it.  This means you'll have to add the character to the `allowed` set here:

https://github.com/scipy/scipy/blob/06f51e32837b4f78f2630701091cd98ff4c6ddf7/tools/unicode-check.py#L13

Reply:
Done thanks.